packages/js/src/settings/routes/site-features.js

```diff
@@ -421,22 +389,19 @@ const SiteFeatures = () => {
 									{ __( "View the XML sitemap", "wordpress-seo" ) }
 									<ExternalLinkIcon className="yst--mr-1 yst-ml-1 yst-h-5 yst-w-5 yst-text-slate-400" />
 								</Button> }
-								<LearnMoreLink id="link-xml-sitemaps" link="https://yoa.st/2a-" />
+								<LearnMoreLink id="link-xml-sitemaps-learn-more" link="https://yoa.st/2a-" ariaLabel={ __( "XML sitemaps", "wordpress-seo" ) } />
```

Comment:
Could you specify this in the PR text so that we can adapt automated tests if needed?

Reply:
Done, I wrote:
Fixed duplicate `id` for `XML sitemaps` feature, changed the `id` of `Learn more` link in this feature to: `link-xml-sitemaps-learn-more`
Anything else I should write them? @igorschoester ?